### DISCUSSION TITLE
Frigate

### DISCUSSION THREAD

**Discussion by freelancers19** [tteck/Proxmox#1644] [Q&A]
Hi everyone.
is it possible to have a script to install Frigate?
Or can someone help me to install it?
Thanks to anyone who will answer me

**Comment by btehan** [comment]
https://github.com/tteck/Proxmox/blob/75635418d2393c7bad714f639cc42d699d3eb211/misc/frigate-support.sh

**Reply by freelancers19** [reply]
how can i use it? I didn't succeed, it gives me errors during the installation.

**Reply by tteck** [reply]
> In the Proxmox web shell run (replace 106 with your LXC ID)
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/frigate-support.sh)" -s 106
Reboot the LXC to apply the changes

**Reply by maxwood** [reply]
What does this script actually do? Install Frigate?

Do I need a Docker LXC setup first before running it?

**Comment by vednolacni** [comment]
> https://github.com/tteck/Proxmox/blob/75635418d2393c7bad714f639cc42d699d3eb211/misc/frigate-support.sh

Does this script works with google coral USB?

**Reply by tteck** [reply]
yes

**Reply by vednolacni** [reply]
Nice! Thank you!

**Comment by freelancers19** [comment]
> > In the Proxmox web shell run (replace 106 with your LXC ID)
> > bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/frigate-support.sh)" -s 106
> > Reboot the LXC to apply the changes

I tried, but I failed

**Comment by tteck** [comment]
Running Frigate on Proxmox
https://www.youtube.com/watch?v=DmbFq5dMsFo

**Reply by jokerigno** [reply]
So your script is no longer supported and you suggest to add docker+frigate instead of a standalone frigate lxc?

**Reply by tteck** [reply]
I've never created a standalone Frigate LXC.

**Reply by jokerigno** [reply]
> I've never created a standalone Frigate LXC.

sorry maybe my question was misleading.

I found this in github: https://raw.githubusercontent.com/tteck/Proxmox/main/misc/frigate-support.sh

So i was expecting to find the script in the website https://tteck.github.io/Proxmox/ but in the Miscellaneous section there's not. So I'm confused.

There's a "tteck way" to install frigate and use it with a coral like scrypted etc?

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/197#discussioncomment-6935150
